oap-server/server-receiver-plugin/skywalking-browser-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/browser/provider/parser/performance/BrowserPerfDataDecorator.java

```diff
@@ -17,98 +17,96 @@
 
 package org.apache.skywalking.oap.server.receiver.browser.provider.parser.performance;
 
-import lombok.RequiredArgsConstructor;
 import org.apache.skywalking.apm.network.language.agent.v3.BrowserPerfData;
 
-@RequiredArgsConstructor
 public class BrowserPerfDataDecorator {
```

Comment:
`BrowserPerfDataDecorator` is redundant now after your changes in this PR

Reply:
Do you ask to change all codes to use builder directly? The builder is always built even before because of `decorator.setTime(nowMillis);` called always.

Reply:
https://github.com/apache/skywalking/blob/8aa046b8d722f0dc40dd6afdf3ce8f1b67894717/oap-server/server-receiver-plugin/skywalking-browser-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/browser/provider/parser/performance/PerfDataAnalyzer.java#L41-L44